### PR TITLE
Add Shufflenetv2 link to main README file

### PR DESCRIPTION
**Pull request by KsenijaS**
# Model Name

## Use-cases
Model use cases

## Description
Description of model

## Model

 |Model        |Download  |Checksum| Download (with sample test data)|ONNX version|Opset version|Accuracy |
|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
|Model Name       |ONNX Model download link with size  | MD5 checksum for the ONNX model| tar file containing ONNX model and synthetic test data (in .pb format)|ONNX version used for conversion| Opset version used for conversion|Accuracy values |

All ONNX models must pass the ONNX model checker before contribution. The snippet of code below can be used to perform the check. If any errors are encountered, it implies the check has failed.

```
import onnx
from onnx import checker
model_proto = onnx.load("path to .onnx file")
checker.check_model(model_proto)
```

<hr>

## Inference
Step by step instructions on how to use the pretrained model and link to an example notebook/code. This section should ideally contain:

### Input to model
Input to network (Example: 224x224 pixels in RGB)

### Preprocessing steps
Preprocessing required

### Output of model
Output of network

### Postprocessing steps
Post processing and meaning of output
<hr>

## Dataset (Train and validation)
This section should discuss datasets and any preparation steps if required.
<hr>

## Validation accuracy
Details of experiments leading to accuracy and comparison with the reference paper.
<hr>

## Training
Training details (preprocessing, hyperparameters, resources and environment) along with link to a training notebook (optional). 

Also clarify in case the model is not trained from scratch and include the source/process used to obtain the ONNX model.
<hr>

## Validation
Validation script/notebook used to obtain accuracy reported above along with details of how to use it and reproduce accuracy.
<hr>

## References
Link to references
<hr>

## Contributors
Contributors' name
<hr>

## License
Add license information
<hr>

### CODE REVIEW

**Comment by KsenijaS** [comment]
@prasanthpul

File: README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ This collection of models take images as input, then classifies the major object
...
-|<b>[ShuffleNet](vision/classification/shufflenet)</b>|[Zhang et al.](https://arxiv.org/abs/1707.01083)|Extremely computation efficient CNN model that is designed specifically for mobile devices. This model greatly reduces the computational cost and provides a ~13x speedup over AlexNet on ARM-based mobile devices. Compared to MobileNet, ShuffleNet achieves superior performance by a significant margin due to it's efficient structure. <br> Top-1 error from paper - ~7.8%|
+|<b>[ShuffleNet_V1](vision/classification/shufflenet)</b>|[Zhang et al.](https://arxiv.org/abs/1707.01083)|Extremely computation efficient CNN model that is designed specifically for mobile devices. This model greatly reduces the computational cost and provides a ~13x speedup over AlexNet on ARM-based mobile devices. Compared to MobileNet, ShuffleNet achieves superior performance by a significant margin due to it's efficient structure. <br> Top-1 error from paper - ~7.8%|
```
Comment (prasanthpul) [inline README.md]: we should remove the top-1 error number for shufflenet v1. in the paper, it says the top-1 error was lowered by 7.8% compared to mobilenet, it does not say the error rate is 7.8%. this issue has been in the description for some time, but we should fix it

File: README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ This collection of models take images as input, then classifies the major object
...
+|<b>[ShuffleNet_V2](vision/classification/shufflenet_v2)</b>|[Zhang et al.](https://arxiv.org/abs/1807.11164)|Extremely computation efficient CNN model that is designed specifically for mobile devices. This network architecture design considers direct metric such as speed, instead of indirect metric like FLOP. <br> Top-1 error from paper - ~30.6%|
```
Comment (prasanthpul) [inline README.md]: from the paper, looks like there are some other lower top-1 error numbers depending on how you run it/measure it. which one is the one that people would expect? if it's not clear, we should probably remove it from the description.

Comment (KsenijaS) [reply README.md]: yes, but in the PyTorch Doc they mention top-1 error as 30.6% https://pytorch.org/hub/pytorch_vision_shufflenet_v2/ and that;s why I chose it.

Comment (vinitra) [reply README.md]: I did a quick search and it looks like 30.6% is the one people would expect -- it's the 1x comparison mentioned in the paper.

File: README.md (hunk 1 above)
Comment (vinitra) [reply README.md]: Prasanth is correct, I think Top-1 error for ShuffleNet is 32.6%
